ui/src/Live/Multiview.tsx

```diff
@@ -165,6 +211,15 @@ const Multiview = (props: MultiviewProps) => {
         },
       }}
     >
+      <Tooltip title="Toggle full screen">
+        <IconButton size="small" sx={{
+          position: 'fixed', background: 'rgba(255,255,255,0.3) !important', transition: '0.2s', opacity: '0.1', bottom: 10, right: 10, zIndex: 9, color: "#fff", ":hover": {
```

Comment:
How would you feel about a bit more opacity? I saw this code, loaded up the interface, and still didn't see this icon the first couple times.

Reply:
Maybe look better than before :)

![image](https://github.com/scottlamb/moonfire-nvr/assets/80969068/aa11f0b5-6c89-41a6-b6d9-8027e0f1a9e3)